### PR TITLE
CO: Added redirect type in AdminImportController

### PR DESCRIPTION
**Pull request by luigimassa**
Added new fileds in AdminImportController to import redirect

| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | added fileds redirect_type and id_type_redirected in AdminImportController
| Type?         | improvement
| Category?     | CO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? |  no
| How to test?  | import csv

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/8345)

### CODE REVIEW

**Comment by mickaelandrieu** [comment]
Hello @luigimassa,

it should work... but I'm wondering why do you need it?

Mickaël

**Comment by luigimassa** [comment]
Hi @mickaelandrieu 
this is the use case 
I created a new products version
I have to deactivate products of previous version, but I don't want lost my seo
So I want to redirect deactivated products to the new products.
With this change I can do a massive update

File: controllers/admin/AdminImportController.php [hunk 1]
```diff
@@ -223,6 +223,8 @@ public function __construct()
                     'id' => array('label' => $this->trans('ID', array(), 'Admin.Global')),
                     'active' => array('label' => $this->trans('Active (0/1)', array(), 'Admin.Advparameters.Feature')),
                     'name' => array('label' => $this->trans('Name', array(), 'Admin.Global')),
+                    'redirect_type' => array('label' => $this->trans('Redirect type', array(), 'Admin.Advparameters.Feature')),
+                    'id_type_redirected' =>   array('label' => $this->trans('Product ID', array(), 'Admin.Advparameters.Feature')),
```
Comment (Quetzacoalt91) [inline controllers/admin/AdminImportController.php]: Be careful, this label is confusing with the ID column and should be something like "Target product / category ID".

**Review by Quetzacoalt91 — CHANGES_REQUESTED** [review]

**Comment by Quetzacoalt91** [comment]
It seems you added by mistake an additional commit to the PR, making the review difficult. Would you mind removing it?

**Comment by Quetzacoalt91** [comment]
Because the Import controller being currently migrated on Symfony (see #11561), the improvement should be brought on the new version when done.

The changes won't be applied to the legacy content, I have no other choice than closing it.

Regards

**Pull request closed without merging**